pepsi

```diff
@@ -1,6 +1,4 @@
-#!/bin/bash
-
-set -e
+#!/usr/bin/env -S bash -e
```

Comment:
The argument for using `/usr/bin/env` is to improve compatibility, right? I've read that [`-S` is non-standard](https://unix.stackexchange.com/a/361795). Would this miss the intention on better compatibility?

Reply:
Thanks for your research and answer. Sounds good to me. Resolving. I will ask how we proceed with merging because you probably don't have the rights for that currently. 😉